operator-framework/src/main/java/io/javaoperatorsdk/operator/api/ResourceController.java

```diff
@@ -28,4 +28,10 @@
      */
     UpdateControl<R> createOrUpdateResource(R resource, Context<R> context);
 
+    // What about we let developer to set the finalizer Name through overriding
+    // this method rather than setting it as attribute in annotation?
+    default String getDefaultFinalizerName() {
```

Comment:
what do you think about setting the finalizer name by overriding this method rather than `Controller.finalizerName` attribute.

Reply:
I'm not a big fan of overriding methods when we just want to set a string. I like to keep it in the annotation.

Reply:
removed this method